ros2action/setup.py

```diff
@@ -43,6 +43,7 @@
             # 'echo = ros2action.verb.echo:EchoVerb',
```

Comment:
I know you didn't add this in, but while we are in here let's remove this comment.  We'll add it back in if/when we implement this.

Reply:
addressed by https://github.com/ros2/ros2cli/pull/894/commits/55f271b502d7cd83130201a807aa4c17d4847444